lib/oci_utils/migrate/imgdevice.py

```diff
@@ -773,11 +773,12 @@ def collect_os_data(self):
             #
             # import operation system type dependant modules
             osrelease = self.get_os_release()
```

Comment:
IT seams that method return a dict , why such cast ?

Reply:
just want to collect the full os release data, and save it in a central repo; currently only need the 'ID', i.e. the flavor of the distro, but extension are in the pipeline, we do not need to look for any release related data again, even if the filesystems, logical volumes, imagefiles are not mounted anymore.

Reply:
Sorry . the spotted line is wrong . My question is about the statement
" bool(osrelease) "

Reply:
just testing if the dictionary is empy, bool() calls the __bool__ ( or in fact __len__) method;
could have been len() > 0 or so, but thought bool() is more clear ....